src/sql/platform/dialog/dialogModal.ts

```diff
@@ -66,12 +66,12 @@ export class DialogModal extends Modal {
 			});
 		}
 
-		this._cancelButton = this.addDialogButton(this._dialog.cancelButton, () => this.cancel(), false);
-		this.updateButtonElement(this._cancelButton, this._dialog.cancelButton);
-		this._dialog.cancelButton.registerClickEvent(this._onCancel.event);
 		this._doneButton = this.addDialogButton(this._dialog.okButton, () => this.done(), false);
```

Comment:
Super-minor: if this is repeated code, how about putting in its own function? 😄

Reply:
I was able to get rid of the `updateButtonElement` line by moving it into the `addDialogButton` function call (which is supposed to address the repetition point you raised already). Unfortunately since the buttons need to be set up to do the done/cancel actions it's hard to combine the remaining lines